wiki/Installation/vi.md

```diff
@@ -0,0 +1,77 @@
+# Cài đặt
+
+*Xem thêm: [Cài đặt trên macOS](/wiki/Installation/macOS)*
+
+Trang này sẽ hướng dẫn bạn làm thế nào để làm osu! hoạt động trên thiết bị Windows của bạn. [osu!academy](/wiki/osu!academy) đã có một [video hướng dẫn](https://www.youtube.com/watch?v=0V5GwzmMhpU) làm thế nào để cài đặt osu! trên Windows.
+
+## Yêu cầu tối thiểu 
+
+- .NET framework 4+ (bộ cài thường sẽ nhắc bạn để cài đặt nó, nếu không thì bạn có thể [tải nó tại](https://www.microsoft.com/en-us/download/details.aspx?id=48130)).
+- Một bộ xử lí đồ họa thường thường.
+
+## Cài đặt osu!
+
+1. Hướng đển [trang tải](https://osu.ppy.sh/home/download) và tải về bộ tải.
+2. Tìm và chạy bộ cài.
+3. Khi bộ cài bắt đầu chạy bạn có 10 giây để chọn vì trí cài đặt.
+   - Nêu bạn muốn đổi vị trí cài đặt, ấn vào dòng chữ phía dưới chứ `Hi!` và chọn vị trí cài đặt.
+   - osu! sẽ được cài đặt vào vị trí đó sau khi đếm ngược hoàn tất. 
+4. Sau khi bộ cài hoàn tất, osu! sẽ xuất hiện. osu!direct sẽ được sử dụng để tải xuống một vài beatmap khởi đầu cho bạn.
+5. osu! sẽ nhắc bạn để đăng nhập hoặc đăng kí.
+   - Nếu bạn đã có tài khoản, đăng nhập. 
+   - Và nếu bạn muốn đăng kí, đọc [Đăng kí](/wiki/Registration).
+
+## Trợ giúp
+
+Nếu bạn đang sử dụng một bản cài đặt 64-bit của Windows 7, bạn có thể gặp một vài vấn để khi cài đặt osu. Hãy thử cài đặt bản 64-bit của .NET Framework, nếu như nó không đi kèm với bộ cài osu!. [Tải nó tại đây](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
+
+---
+
+**Nếu như bộ cài đặt thông báo lỗi khi bạn cài đặt, bạn có thể thử những điều sau:**
+
+- Hãy thử cài đặt phiên bản mới nhất của .NET Frameword. [Bạn có thể tìm thấy nó tại đây](https://dotnet.microsoft.com/download).
+- Dọn PC của bạn - Quét máy với phần mềm diệt virus, dọn registry. Một phần mềm thường được giới thiệu để dọn dẹp là CCleaner. Quét cả files trên máy tính lẫn registry của bạn. Bộ cài cũng có thể gặp phải lỗi nếu như registry vẫn còn giá trị của một phiên bản osu! trước đó trong thiết bị của bạn.
+
+## Thêm Beatmaps
+
+*Xem thêm: [Beatmaps](/wiki/Beatmap)*
+
+Có 2 cách để thêm beatmaps vào game, qua website hoặc sử dụng osu!direct (yêu cầu [osu!supporter](/wiki/osu!supporter)).
+
+### Website
+
+1. Hướng tới [trang danh sách beatmap](https://osu.ppy.sh/beatmapsets) và đăng nhập vào webs
+2. Tìm bài hát mà bạn muốn chơi.
+3. Ấn vào nút tải xuống và lưu beatmap về máy tính của bạn.
+4. Tìm và mở beatmap file, nó có đuôi `.osz`.
```

Comment:
```suggestion
4. Tìm và mở tệp beatmap, nó có đuôi `.osz`.
```
If you don't want to translate terminologies, you should still use `file beatmap` instead of `beatmap file` in Vietnamese.